pkg/pillar/types/assignableadapters.go

```diff
@@ -88,11 +88,31 @@ type IoBundle struct {
 	KeepInHost bool
 	Error      string
 	ErrorTime  time.Time
+
+	VFs VFList
 }
 
 // Really a constant
 var nilUUID = uuid.UUID{}
 
+func (ib *IoBundle) GetPciLong() string {
+	if ib.Type == IoNetEthVF {
+		if vf := ib.VFs.UsedBy(ib.UsedByUUID); vf != nil {
+			return vf.PciLong
+		}
+	}
+	return ib.PciLong
+}
+
+func (ib *IoBundle) GetMacAddr() string {
+	if ib.Type == IoNetEthVF {
+		if vf := ib.VFs.UsedBy(ib.UsedByUUID); vf != nil {
```

Comment:
Does this mean there can only be one VF on a particular PF assigned to the same app instance?
That might not be sufficiently flexible.